test_plugin/README.md

```diff
@@ -0,0 +1,130 @@
+# Plugins
+
+Short explanation showing how Deno Rust plugins can be used. _**!!! warning the
+feature is still unstable!!!**_
+
+Clone the [deno](https://github.com/denoland/deno.git) source & go to
+`deno/test_plugin`
+
+The repo is structure as following:
+
+```
+test_plugin
+├── Cargo.toml
+├── src
+│   └── lib.rs
+└── tests
+    ├── ...
+    └── test.js
+```
+
+It is a rust repo.
+
+## In the `rust`
+
+#### 1- add `deno_core` & specify your project as a crate of type `cdylib` to make the lib dynamically available.
+
+In the cargo manifest
+
+```
+[package]
+name = "test_plugin"
+...
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+...
+
+deno_core = { path = "../core" }
```

Comment:
This is not how you'd usually add `deno_core`.

Reply:
yep, outside the deno project we rely on the Rust community’s crate registry! Change done